website/functions.js

```diff
@@ -0,0 +1,32 @@
+const readAloud = function(rowNum) {
+  rowNum = rowNum.toString();
```

Comment:
Let's give each row an ID, and then select "#row0 td.char" which will give us 4 cells. That way, we will be able to remove the IDs for each cell.